### ISSUE TITLE
Update documentation and package references

### ISSUE THREAD

**Issue by warmfusion** [freshworks/redis-operator#15]
### Expected behaviour

When using a forked version of the redis-operator from the `freshworks` group, I expect documentation and package references to reflect the forked variation of the project.

### Actual behaviour

Everything remains refering to spotahome

### Steps to reproduce the behaviour

Read https://github.com/freshworks/redis-operator?tab=readme-ov-file#operator-deployment-on-kubernetes

**Comment by samof76** [comment]
We have still not heard from the spotahome folks. So we will make these changes soon.

**Comment by pascaliske** [comment]
GitHub Pages also needs to be configured using a branch (`gh-pages`) based workflow to replace the URL for the Helm chart:

https://spotahome.github.io/redis-operator → https://freshworks.github.io/redis-operator.

Until that setting is enabled the latest chart release (`Chart-3.3.1`) remains inaccessible for Helm. 🙈

**Comment by ab-trusty** [comment]
Yeah we're trying to migrate from spotahome to your fork but are finding issues with helm and the chart not being available
Thanks for the efforts, if we can fix it or make a pr please just say so

**Comment by pascaliske** [comment]
Looks like #46 solves this finally. 🙂
